mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -22,7 +27,13 @@ class StatefulHomeTest {
     val api = AuthenticationFacade(emptyAuth(), emptyStore())
     api.signUpWithEmail("email", "name", "password")
     val user = api.currentUser.filterIsInstance<AuthenticatedUser>().first()
-    val strings = rule.setContentWithLocalizedStrings { StatefulHome(user) }
+    val mockSocialFacade = mockk<SocialFacade>()
+    every { mockSocialFacade.search("") } returns emptyFlow()
+    val mockAuthenticationFacade = mockk<AuthenticationFacade>()
+    val strings =
+        rule.setContentWithLocalizedStrings {
+          ProvideFacades(mockAuthenticationFacade, mockSocialFacade) { StatefulHome(user) }
+        }
     rule.onNodeWithText(strings.sectionSocial).assertIsSelected()
     rule.onNodeWithText(strings.sectionSettings).assertIsNotSelected()
   }
```

Comment:
I think we should be using the in-memory `Auth` and `Store` to create the `AuthenticationFacade` and `SocialFacade` rather than mocks.